tests/unit/components/NcAppSidebar/NcAppSidebarTabs.spec.js

```diff
@@ -83,7 +87,7 @@ describe('NcAppSidebarTabs.vue', () => {
 					},
 				})
 			})
-			it('Issues no warning.', () => {
+			it('Issues no warning.', (onWarning) => {
```

Comment:
Shouldn't it be `({ onWarning })`?

Reply:
Could be. But these tests don't run anyway at the moment, because the underlying component is not yet migrated to vue3. So these adjustments are rather placeholders, than real fixes yet.